ports/esp32/machine_uart.c

```diff
@@ -68,28 +70,28 @@ STATIC void machine_uart_print(const mp_print_t *print, mp_obj_t self_in, mp_pri
     if (self->invert) {
         mp_printf(print, ", invert=");
         uint32_t invert_mask = self->invert;
-        if (invert_mask & UART_INVERSE_TXD) {
+        if (invert_mask & UART_SIGNAL_TXD_INV) {
```

Comment:
Does this work with esp-idf pre-4.1? Or is the idea to deprecate the use of older esp-idf versions?

Reply:
I didn't get that far, I just wanted to get something working and get feedback (will add WIP to the PR title).

We can continue to support older IDF versions if it's not difficult, but I'd rather just support the latest (easier to test, etc).

Reply:
NP, I just wanted to flag it.